ui/src/setupTests.js

```diff
@@ -8,13 +8,13 @@ require('mutationobserver-shim');
  */
 const enzyme = require("enzyme");
 const Adapter = require("enzyme-adapter-react-16");
+const reactDOM = require("react-dom");
```

Comment:
this appears to be unused

Reply:
no change when I comment this out

Reply:
Oops, thanks. This was leftover from the previous approach